source/basic.tex

```diff
@@ -2233,8 +2233,8 @@
 
 \pnum
 \indextext{program}%
-A \defn{program} consists of one or more \defn{translation
-units} (Clause~\ref{lex}) linked together. A translation unit consists
+A \defn{program} consists of one or more \defn{translation units}
```

Comment:
Do we really need a separate entry for "translation units" (plural)? Can we not index this also under "translation unit"?